internal/driver/config.go

```diff
@@ -28,17 +26,25 @@ type driverConfiguration struct {
 	MaxDiscoverDurationSeconds int
 }
 
-var (
-	// defaultConfig holds default values for each configurable item in case
-	// they are not present in the configuration
-	defaultConfig = map[string]string{
-		"DiscoverySubnets":           "",
-		"ProbeAsyncLimit":            "5000",
-		"ProbeTimeoutSeconds":        "2",
-		"ScanPort":                   "5084",
-		"MaxDiscoverDurationSeconds": "300",
+type ServiceConfig struct {
```

Comment:
add doc comment above as it is package public

Reply:
Comment has been added